src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -62,8 +65,12 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
         try {
             const { username, password } = await this.getCredentials(false, true);
 
-            const sslString: string | undefined = process.env.POSTGRES_SSL;
-            const ssl: boolean = sslString === 'true';
+            const ssl: ConnectionOptions = {
+                // Always provide the certificate since it is accepted even when SSL is disabled
+                // Certificate source: https://aka.ms/AA7wnvl
+                ca: fse.readFileSync(path.join(getResourcesPath(), 'certificates', 'BaltimoreCyberTrustRoot.crt.pem')).toString()
```

Comment:
Always use the async version when possible. `await fse.readFile` instead of `fse.readFileSync`